app/views/clients/_match_results.html.haml

```diff
@@ -14,7 +14,7 @@
           - sc_count = c.source_clients.count
           - colspan = if sc_count == 1 then 2 else 4 end
           - if sc_count == 1
-            - client_name = "#{client_name} in #{c.source_clients.first.data_source.short_name}"
+            - client_name = "#{client_name} in #{c.source_clients.first.data_source&.short_name}"
```

Comment:
Don't include 'in' if there is no datasource?

Reply:
We could.  These should never be without a data source in production, but I have some bad data locally, and this allowed me to use the page.

Reply:
Seems like unnecessary work.